test/features/app/app.js

```diff
@@ -2,6 +2,7 @@
 
 
 var preq   = require('preq');
+var rp     = require('request-promise');
```

Comment:
`preq` is a promisified request version used throughout all the services

Reply:
When content is gzipped, `preq` unzips it and removes the `content-encoding` header. It cannot be used to test that content is returned gzipped.

Reply:
try providing `accept-encoding: gzip` header to the request, I think if that's provided it will not unzip the content, but I'm unsure.

Reply:
If you look at the test `it('should get static content gzipped'), ...`, it adds `accept-encoding: 'gzip, deflate'`. It was skipped because it failed for a while. 

https://github.com/wikimedia/preq/commit/55e2d285c310dae302907e283def913d55292b0e